exercise/cache/exercise.py

```diff
@@ -1,10 +1,11 @@
-import time, lzma
+import time
 from django.conf import settings
 from django.db.models.signals import post_save, post_delete
 
 from lib.cache import CachedAbstract
 from lib.remote_page import RemotePageNotModified
 from ..protocol.aplus import load_exercise_page
+from lzma import compress as compressor, decompress as decompressor
```

Comment:
Good to know, thanks! I'll change that.

Reply:
So to my understanding the line should be after ```import time``` and there should also be a blank line before django imports because django is not a standard module, is that correct?

Reply:
Yes. The correct order and newlines are not used in many places, but that should be fixed when we edit the imports. I also forgot to mention that all groups should be sorted alphabeticall e.g.:

```
import os
from lzma import ..
from time import time
from zlib import ...
```